mkosi

```diff
@@ -4285,9 +4264,12 @@ def find_cache(args: CommandLineArguments) -> None:
 def require_private_file(name: str, description: str) -> None:
     mode = os.stat(name).st_mode & 0o777
     if mode & 0o007:
-        warn("Permissions of '{}' of '{}' are too open.\n" +
-             "When creating {} files use an access mode that restricts access to the owner only.",
-             name, oct(mode), description)
+        warn(
+            f"""\
+            Permissions of '{name}' of '{oct(mode)}' are too open.
```

Comment:
```suggestion
            Permissions of '{name}' of '{mode:04o}' are too open.
```
It's a change, but it will print proper four-digit permissions without the `0b` prefix.